### ISSUE TITLE
Regional Regulation Request by Reference

### ISSUE THREAD

**Issue by XTechnology-TR** [YunsGeek/Quraan-Player#5]
Hello, first of all I would like to congratulate you for a successful application. You will see the reward on both sides of making it easier to listen to the word of Allah, who is more precious and valuable than our lives, insha'Allah.

Other than that, I don't think you will have any expectations. Unfortunately, sometimes some similar ap, book, etc., for a fee or for worldly benefits. I see it has been sold. Personally, I think it's a very wrong approach.

In short, I am interested in forking, translating into Turkish, making minor changes according to the geographical characteristics of the people of the region and sharing them. Are there any requirements or different negativities for this subject?

**Comment by YunsGeek** [comment]
hello , thank you brother , you can do what every you like , the original Quran comes for free and we can share it with people .

i hope you make the app easier for your region , god bless you .
thank you again , have a nice day .
